pkg/cmd/root.go

```diff
@@ -95,16 +95,7 @@ By default, the default directories depend on the Operating System. The defaults
 `
 
 var settings = cli.New()
-
-func Debug(format string, v ...interface{}) {
-	if settings.Debug {
-		log.Output(2, fmt.Sprintf("[debug] "+format+"\n", v...))
-	}
-}
-
-func Warning(format string, v ...interface{}) {
-	fmt.Fprintf(os.Stderr, "WARNING: "+format+"\n", v...)
-}
+var Logger = cli.NewLogger(settings.Debug)
```

Comment:
Not sure if this is what we want with how messages are rendered with slog [TextHandler](https://pkg.go.dev/golang.org/x/exp/slog#TextHandler). We have very few `key=value` with `Logger.Debug` or `Logger.Warn`, so I think it's ok.

Reply:
As with the last commits. https://github.com/helm/helm/pull/30708/commits/e7eedae97cb8ea9050ec220a7cb33445c0b791f9

```
benoit.tigeot » bin/helm upgrade --install --debug --wait frontend \         
--namespace test \
--set replicaCount=2 \
--set backend=http://backend-podinfo:9898/echo \
podinfo/podinfo
level=DEBUG msg="getting history for release" release=frontend
level=DEBUG msg="getting release history" name=frontend
level=DEBUG msg="preparing upgrade" name=frontend
```